.pre-commit-config.yaml

```diff
@@ -21,4 +21,4 @@ repos:
     rev: 22.10.0
     hooks:
       - id: black
-        language_version: python3.9
+        language_version: python3
```

Comment:
we shouldn't be changing this.  I saw Tim do this as well

Reply:
Updated it! Thank you!